fighthealthinsurance/medicaid_gov_api.py

```
@@ -0,0 +1,499 @@
+
+    The declaration scan covers the WHOLE document, not a prefix of it: a
+    long enough leading comment pushes ``<!DOCTYPE`` past any fixed window,
+    and a guard you can walk around with padding is not a guard. The size cap
+    runs first, so the scan is bounded.
+    """
+    if len(content) > _MAX_SITEMAP_BYTES:
+        raise ValueError(f"sitemap document too large ({len(content)} bytes)")
+    lowered = content.lower()
+    if b"<!doctype" in lowered or b"<!entity" in lowered:
+        raise ValueError("sitemap document carries a doctype/entity declaration")
+    return ET.fromstring(content)
```

Ruff (0.16.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)